src/components/Checkout/InContext/InContextCheckout.vue

```diff
@@ -96,7 +101,9 @@
 			</kv-grid>
 
 			<kiva-credit-payment
-				v-if="showKivaCreditButton && !promoGuestCheckoutEnabled"
+				v-if="showKivaCreditButton
+					&& !promoGuestCheckoutEnabled
```

Comment:
@michelleinez Testing this locally and I think this line should be `!continueAsGuest` instead otherwise an existing user on a guest checkout enabled account wouldn't be able to see the button.

Reply:
I realize it's prob fine for now as `promoGuestCheckoutEnabled` is gonna be false most cases, I can just add to my current work.